README.md

```diff
@@ -1,7 +1,8 @@
 # ISAR
 
-ISAR - Integration and Supervisory control of Autonomous Robots - is a tool for
-integrating robot applications into Equinor systems. Through the ISAR API you can send command to a robot to do missions and collect results from the missions.
+ISAR - Integration and Supervisory control of Autonomous Robots - is a tool for integrating robot applications into
+Equinor systems. Through the ISAR API you can send command to a robot to do missions and collect results from the
+missions.
```

Comment:
Any special reason to split the sentences?

Reply:
They were split because formatter did so automatically. Before that the entire sentence went "out-of-bounds" and I had to scroll back and forth in the README.